### PR TITLE
CTONET-971 + CTONET-974

### PR DESCRIPTION
**Pull request by isaacdorfman**
Added nodeSelector/nodeAffinity and testGracePeriod to the documentation

### CODE REVIEW

**Comment by crwr45** [comment]
Please rebase and resolve conflicts

File: README.md [hunk 1]
```diff
@@ -128,6 +128,8 @@ iterations may consider running `lshw` or similar types of diagnostic tests.
 The `generic` test spec tests:
 1) `Default` network connectivity between containers.
 2) That CNF container images are RHEL based.
+3) Print a warning if a pod spec specifies nodeSelector/nodeAffinity
+4) Print a warning if a pod's terminationGracePeriod is set to 30s(the default in Kubernetes)
```
Comment (ryandgoulding) [inline README.md:132]: Please replace `Kubernetes` with `OpenShift`.

**Review by ryandgoulding — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -128,6 +128,8 @@ iterations may consider running `lshw` or similar types of diagnostic tests.
 The `generic` test spec tests:
 1) `Default` network connectivity between containers.
 2) That CNF container images are RHEL based.
+3) Print a warning if a pod spec specifies nodeSelector/nodeAffinity
```
Comment (ryandgoulding) [inline README.md:131]: Please end the sentence with a period.

**Comment by isaacdorfman** [comment]
@ryandgoulding @crwr45 Fixed and rebased

**Review by crwr45 — APPROVED** [review]

**Pull request merged**